picobot/painter.py

```diff
@@ -222,9 +224,18 @@ def sticker_from_text(
     if limit_is_user:
         bigger_size = title_size
     else:
-        if len(text) >= 26:
-            aux_text = wrapped_text(text, line_limit=26)
+        if len(text) >= LINE_WIDTH_LIMIT:
+            line_width = LINE_WIDTH_LIMIT
+            aux_text = wrapped_text(text, line_width=line_width)
             text_size = aux_img.multiline_textsize(aux_text, font=font)
+            # Try to make a better width/height ratio for long messages
+            for i in range(3):
```

Comment:
Small nit: as `i` is not being used, you can leave it as an `_`:

```suggestion
            for _ in range(3):
```